geometric_kernels/kernels/feature_maps.py

```diff
@@ -418,4 +338,54 @@ def _map(
 
         return key, features
 
-    return _map
+
+# compatibility layer
+def deprecated(fn):
+    """
+    Ad-hoc decorator that suggests using new camel-cased classes instead of
+    old snake-cased functions.
+    """
+    fn_name = fn.__name__
+    new_name = "".join(word.title() for word in fn_name.split("_"))
+
+    doc = globals()[new_name].__init__.__doc__
+
+    def wrapper(*args, **kwargs):
+        warnings.simplefilter("always", DeprecationWarning)  # turn off filter
```

Comment:
Isn't the user supposed to decide which warnings they want to see or not? Changing filtering in our code seems weird.